azure-pipelines.yml

```diff
@@ -189,16 +189,17 @@ jobs:
 
       # Run example applications
       for f in doc/examples/*/*.py; do
-        if [[ "${f}" == *"plot_3d_interaction"* ]]; then
-          echo "skipped"
-        else
-          $PYTHON -W ignore:Matplotlib:UserWarning "${f}"
-          if [ $? -ne 0 ]; then
-            exit 1
-          fi
+        # Comment out any plotly.io.show() calls before running the example.
+        # Plotly opens a web browser, which often seemed to cause the CI to
+        # hang and timeout.
+        sed -i 's/plotly.io.show/# plotly.io.show/g' ${f}
+
+        $PYTHON -W ignore:Matplotlib:UserWarning "${f}"
+        if [ $? -ne 0 ]; then
+          exit 1
         fi
       done
-    condition:
+    condition: eq(variables['TEST_EXAMPLES'], 'true')
```

Comment:
I guess this will disappear upon merging `main`

Reply:
yeah, it does not show up in the diff anymore. this got modified unintentionally during the recent CI-based benchmarking PR, but we later restored it.